aggregate/load_aggregated.py

```diff
@@ -69,3 +69,37 @@ def initialize_dataframe_geo_index(geography, columns=[]):
     rv = pd.DataFrame(index=indicies[geography], columns=columns)
     rv.index.rename(geography, inplace=True)
     return rv
+
+
+"""this is specifically to use for housing security and quality March 4th POP data"""
+def load_clean_pop_data(ind_name_str: str) -> pd.DataFrame:
```

Comment:
this function should def be renamed to `load_clean_QOL_pop_data` or something similar.

Reply:
wait, I mean `load_clean_housing_security_dcp_pop`. Category here is housing security. What is docstring trying to say about QOL indicators?